docs/topics/execute.adoc

```diff
@@ -68,6 +68,49 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 $ __RHAMT_HOME__/bin/rhamt-cli -Dwindup.decompiler=procyon --input __INPUT_ARCHIVE_OR_DIRECTORY__ --output __OUTPUT_REPORT_DIRECTORY__ --target __TARGET_TECHNOLOGY__ --packages __PACKAGE_1__ __PACKAGE_2__
 ----
 
+[discrete]
+[[cli_bash_completion]]
+== {ProductShortName} {CLINameTitle} Bash Completion
+
+The {ProductShortName} {CLIName} includes a set of bash completions, allowing the xref:command_line_arguments[{ProductShortName} Command-line Arguments] to be auto completed by pressing Tab when entering the commands. For instance, entering the following displays a list of available arguments.
```

Comment:
- I think that "Command-line Arguments" in the link should be lower case since you're using the link in a running sentence, not saying that this is the title of the section.
- I'm wondering if "Tab" should be in bold or somehow marked up to indicate that it's different than a normal word. I'm not sure if we have a standard for keyboard keys (on EAP at least).

Reply:
* I've adjusted the link to be lowercase.
* I'm not certain how to display this - I spoke with Sande, and I believe we use "Ctrl", with the first letter uppercase, but I didn't see tab used anywhere.